src/common/ownsql.h

```diff
@@ -136,10 +136,15 @@ class OCSYNC_EXPORT SqlQuery
     template <class T>
     void bindValue(int pos, const T &value)
     {
-        QString s;
-        QDebug(&s).noquote().nospace() << '\'' << value << '\'';
-        _boundQuery.replace(QStringLiteral("?%1").arg(QString::number(pos)), s);
-        bindValueConvert(pos, value);
+        const auto converted = convertValue(value);
+        if (lcSql().isDebugEnabled()) {
```

Comment:
Is this some sort of runtime optimization? First time I see such an if in the code base.